cubic-server/Chat.hpp

```diff
@@ -21,7 +21,7 @@ class Chat {
      * @param message
      * @param from
      */
-    void sendPlayerMessage(const chat::Message &message, const Player &from);
+    void sendPlayerMessage(const chat::Message &message, const std::shared_ptr<Player> from);
```

Comment:
Why did you change everything reference to the player to shared_ptrs?

Reply:
I remember getting angry over this, but I don't remember why...

Reply:
I can't remember, I am reversing this